jobs/component_jobs.groovy

```diff
@@ -121,11 +121,18 @@ repos.each { Map repo ->
           '''.stripIndent().trim()
         }
         // do not run e2e tests for workflow-manager at this time
-        if (repo.name != 'workflow-manager') {
+        // also don't run if author is 'Jenkins User' as this is from a ci
+        // increment commit which would have already been tested
+        if (repo.name != 'workflow-manager' || '${GIT_COMMITTER_NAME}' != 'Jenkins User' ) {
```

Comment:
Realizing this needs to be in `workflow_test.groovy` and not here.  Consider the scenario where a `component` job is run on a commit to the master branch of that component.  If the downstream test job is successful, we auto bump the image tag version in the chart and commit that.  Therefore, the `workflow-test` job, which watches commits in the `charts` repo will run e2e _again_  (triggered by the auto bump commit) using a change that has already been vetted.